app/views/idv/inherited_proofing/no_information.html.erb

```diff
@@ -0,0 +1,25 @@
+<%= render(
+      'idv/shared/error',
+      type: :warning,
+      title: t('inherited_proofing.errors.cannot_retrieve.title'),
+      heading: t('inherited_proofing.errors.cannot_retrieve.heading'),
+    ) do %>
+  <p>
+     <%= t('inherited_proofing.errors.cannot_retrieve.info') %>
+  </p>
+
+  <%= link_to t('inherited_proofing.buttons.try_again'), root_url, class: 'usa-button usa-button--big usa-button--wide' %>
+
+  <%= render(
+        'shared/troubleshooting_options',
+        heading_tag: :h3,
```

Comment:
We should leave out this option so that the default `:h2` heading is used, since otherwise there's an accessibility issue, as the heading order increases from `h1` to `h3`.

https://dequeuniversity.com/rules/axe/3.5/heading-order

```suggestion
```